flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/api/TableConfigOptions.java

```diff
@@ -151,6 +151,11 @@
 					.defaultValue(32)
 					.withDescription("Sets the table reserved memory size of hashAgg operator. It defines the lower limit.");
 
+	public static final ConfigOption<Integer> SQL_RESOURCE_HASH_JOIN_TABLE_MEM =
+			key("sql.resource.hash-join.table.memory.mb")
+					.defaultValue(32)
```

Comment:
seems too small

Reply:
Now hash agg and sort default memory is 32M too.